modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -0,0 +1,116 @@
+[[exportword-service-url]]
+== `exportword_service_url`
+
+The {pluginname} uses the {productname} Cloud Services HTML to DOCX converter service to generate the Word files.
+
+NOTE: The {pluginname} feature is currently only available `on-premise` and requires the `exportword_service_url` option to be configured. https://www.tiny.cloud/contact/[Contact us] if you require this feature.
+
+*Type:* `+String+`
+
+=== Example: using `exportword_service_url`
+
+// Add a working and tested configuration.
```

Comment:
Does anything need to be done to address this comment?

Reply:
```suggestion
```